lib/resolve-value.js

```diff
@@ -27,26 +29,61 @@ function toString(value) {
 var resolveValue = function(decl, map, /*optional*/ignorePseudoScope, /*internal debugging*/_debugIsInternal) {
 	var debugIndent = _debugIsInternal ? '\t' : '';
 
+	var matchingVarDecl = undefined;
+	var RE_VAR_FUNC_G = new RegExp(RE_VAR_FUNC.source, 'g');
```

Comment:
We can get rid of `RE_VAR_FUNC_G` and `RE_VAR_FUNC` now

Reply:
`RE_VAR_FUNC_G` was removed.

But, It seems we can't remove `RE_VAR_FUNC`, since it is being called in `index.js`. 

https://github.com/MadLittleMods/postcss-css-variables/blob/5695b77f5b7b1bca7f570ef1b1fb324fc0f432f7/index.js#L216-L227